Eshop.WebApi/Controllers/ProductController.cs

```diff
@@ -7,6 +8,13 @@ namespace Eshop.WebApi.Controllers
     [Route("[controller]")]
     public class ProductController : Controller
     {
+        private readonly EshopDbContext dbContext;
+
+        public ProductController(EshopDbContext dbContext)
+        {
+            this.dbContext = dbContext;
+        }
+
         private static List<Product> Products = new List<Product>
```

Comment:
now this should be not needed - remove it and fix it if there is some usage

Reply:
removed